src/app/pages/SpotTradingPage/components/TradeForm/MarketForm.tsx

```diff
@@ -0,0 +1,295 @@
+  );
+
+  return (
+    <div className={classNames({ 'tw-hidden': hidden })}>
+      <TradeDialog
+        onCloseModal={() => setIsTradingDialogOpen(false)}
+        isOpen={isTradingDialogOpen}
+        slippage={slippage}
+        tradeType={tradeType}
+        orderType={OrderType.MARKET}
+        amount={amount}
+        minReturn={minReturn}
+        targetToken={targetToken}
+        sourceToken={sourceToken}
+        expectedReturn={weiToFixed(rateByPath, 6)}
+        submit={send}
+        fee={
+          <TxFeeCalculator
+            args={[
+              getTokenContract(sourceToken).address,
+              getTokenContract(targetToken).address,
+              toWei(amount || '0'),
+            ]}
+            methodName="getSwapExpectedReturn"
+            contractName="sovrynProtocol"
+            className="tw-mb-0"
+          />
+        }
+      />
+      <TransactionDialog
+        tx={{ ...tx, retry: send }}
+        onUserConfirmed={() => setIsTradingDialogOpen(false)}
+        onSuccess={() => showToast('success')}
+        onError={() => showToast('error')}
+        action={t(translations.spotTradingPage.tradeDialog.order)}
+        fee={
+          <TxFeeCalculator
+            args={[
+              getTokenContract(sourceToken).address,
+              getTokenContract(targetToken).address,
+              toWei(amount || '0'),
+            ]}
+            methodName="getSwapExpectedReturn"
+            contractName="sovrynProtocol"
+          />
+        }
+        finalMessage={
+          <OrderView
+            tradeType={tradeType}
+            orderType={OrderType.MARKET}
+            amount={amount}
+            minReturn={minReturn}
+            targetToken={targetToken}
+            sourceToken={sourceToken}
+            expectedReturn={weiToFixed(rateByPath, 6)}
+          />
+        }
+      />
+
+      <div className="tw-mx-auto">
+        <div className="tw-mb-2 tw-mt-2 tw-bg-gray-5 tw-py-2 tw-text-center tw-flex tw-items-center tw-justify-center tw-rounded-lg">
+          <AvailableBalance
+            className="tw-mb-0 tw-justify-center"
+            asset={sourceToken}
+          />
+        </div>
+        <div className="tw-flex tw-items-center tw-justify-between tw-mt-5">
+          <span className={styles.amountLabel + ' tw-mr-4'}>Amount:</span>
+          <AmountInput
+            value={amount}
+            onChange={value => setAmount(value)}
+            asset={sourceToken}
+            hideAmountSelector
+            dataActionId="spot-market-amountInput"
+          />
+        </div>
+
+        <FormGroup
+          className="tw-px-3 tw-mt-8"
+          label={t(translations.buySovPage.slippageDialog.tolerance)}
+        >
+          <Slider
+            value={slippage}
+            onChange={setSlippage}
+            min={0.1}
+            max={1}
+            stepSize={0.05}
+            labelRenderer={value => <>{value}%</>}
+            labelValues={[0.1, 0.25, 0.5, 0.75, 1]}
```

Comment:
I wrote it 2 times already so probably extract it to some file and name it e.g. `sliderDefaultLabelValues`. 

Or even better - create some slider config file and extract this + `stepSize`, `max` and `min` there and use it in all places where it's hardcoded now.

Reply:
done